### DISCUSSION TITLE
Latest Bitwarden client 1.31.0 no longer appears as a premium account

### DISCUSSION THREAD

**Discussion by Netrvin** [dani-garcia/vaultwarden#2297] [General]
### Subject of the issue

Latest Bitwarden client 1.31.0 no longer appears as a premium account
Although attachments and TOTP functions are not affected, it will prompt you not to be a premium account when operating attachments.

### Deployment environment

* vaultwarden version: 1.24.0

* Install method: Docker image

* Clients used: Desktop(Windows) 1.31.0

* Reverse proxy and version: Nginx

* MySQL/MariaDB or PostgreSQL version: No

* Other relevant details: No

### Steps to reproduce

Click Account -> Premium Membership

### Expected behaviour

Appears as a premium account as before

### Actual behaviour

"You are not currently a premium member." appeared.

### Troubleshooting data

![Snipaste_2022-02-11_20-40-54](https://user-images.githubusercontent.com/11222509/153593131-00f9f709-012d-44f5-9475-77b01282f5d2.png)

**Comment by BlackDex** [comment]
I'm not able to reproduce this. You are sure you are logged-in into a Vaultwarden environment?

**Comment by Netrvin** [comment]
> I'm not able to reproduce this. You are sure you are logged-in into a Vaultwarden environment?

Yes
![Snipaste_2022-02-11_21-01-11](https://user-images.githubusercontent.com/11222509/153596276-c2092700-307f-4148-9a76-991ac9f3f280.png)

 I used devtools to check the request made when the refresh button on the premium account panel is pressed, the following is the response, is there any problem?
```
{"Kdf":0,"KdfIterations":100000,"Key":"xxx","PrivateKey":"xxx","ResetMasterPassword":false,"access_token":"xxx","expires_in":7200,"refresh_token":"xxx","scope":"api offline_access","token_type":"Bearer","unofficialServer":true}
```

**Comment by Netrvin** [comment]
I try to download the client from the official website again. And the version is 1.30.0. It remains 1.30.0 after checking update. So...I guess I was updated with a new version of the client that has been withdrawn? It feels weird...
The problem doesn't appear in 1.30.0.

**Comment by BlackDex** [comment]
I have version v1.31.0, and no issues with that.
But, it looks like it is a Bitwarden Desktop issue: https://github.com/bitwarden/desktop/issues/1320
Nothing we can do on our side i think.
